p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1176,7 +1176,7 @@ produced by different programmers at different times:
 Similar to the C standard library, the P4 language specification
 defines a core library that declares useful P4 constructs. A
 description of the P4 core library is provided in Appendix
-[#sec-p4-core-lib]. Most P4 programs will include the core
+[#sec-p4-core-lib]. All P4 programs must include the core
```

Comment:
Sometimes the compiler will synthesize code using verify even if you don't use it in your program. In this case if you haven't included core.p4 you will get very mysterious errors. So it should be "must".